scripts/version_calc.py

```
@@ -0,0 +1,312 @@
+    PATCH = 1
+    MINOR = 2
+    MAJOR = 3
+
+
+@dataclass(frozen=True)
+class Version:
+    major: int
+    minor: int
+    patch: int
+
+    def bump(self, level: BumpLevel) -> "Version":
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)